manifests/l/libvips/libvips/8.17.3/libvips.libvips.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created with WinGet Updater using komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: libvips.libvips
+PackageVersion: 8.17.3
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: vips-dev-8.17/bin/vips.exe
+  PortableCommandAlias: vips
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2025-10-30
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/libvips/build-win64-mxe/releases/download/v8.17.3/vips-dev-w64-all-8.17.3.zip
+  InstallerSha256: 0A57AAC04CA723DEF28BA607EC1EF53B984A52594EE3E13D0E7D8EA89455A357

```

```suggestion
  InstallerUrl: https://github.com/libvips/build-win64-mxe/releases/download/v8.17.3/vips-dev-w32-all-8.17.3.zip
  InstallerSha256: 4A45A552665DF627633CC8C94AC313BA465D0DB824E0CA6AC0E436D7D944BDE2
```

FWIW, version 8.17.2 also needs to be corrected:
https://github.com/microsoft/winget-pkgs/blob/e7bab93757532cccbd30e43b1decc2e1730247c2/manifests/l/libvips/libvips/8.17.2/libvips.libvips.installer.yaml#L18-L20